content/docs/ops/maintenance-list.md

```diff
@@ -29,7 +29,7 @@ that week.
 
 - Confirm that all AWS users have MFA configured. The following should return `[]`
 or the users that don't have MFA enabled:
-```shell
+```bash
```

Comment:
It looks like we use `shell`, `sh`, and `bash` a bunch throughout this repo. WDYT about picking one and standardizing?